app/models/loan.rb

```diff
@@ -64,6 +70,8 @@ class Loan < ActiveRecord::Base
   attr_option_settable :status, :loan_type, :project_type, :public_level
 
 
+  attr_custom_model_linkable LOAN_CRITERIA_FIELD_SET_SYMBOL, POST_ANALYSIS_FIELD_SET_SYMBOL
```

Comment:
I'm not sure if I understand the reasoning behind not just using `attr_custom_model_linkable :loan_criteria, :loan_post_analysis` here.